client/src/pages/_app.js

```diff
@@ -3,13 +3,14 @@ import { Grommet } from 'grommet'
 import Head from 'next/head'
 import theme from '../theme'
 import Layout from '../components/Layout'
+import { ResourcesPortalContextProvider } from '../ResouresPortalContext'
```

Comment:
```suggestion
import { ResourcesPortalContext } from '../ResourcesPortalContext'
```
This will necessitate a file change as well.

Reply:
good catch i updated the file and this in a single commit!